experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -12,7 +12,24 @@ import { ChangesetLocalId, NodeChangeset } from "../modular-schema";
 export type ProtoNode = JsonableTree;
 
 export type NodeCount = number;
-export type Skip = number;
+
+/**
+ * Left undefined for terseness.
+ */
+export const SkipType = undefined;
+
+export interface Skip extends CellTargetingMark {
```

Comment:
What do you think about renaming `Skip` to something like `UnchangedMark` or `PlaceholderMark`?

Reply:
How about `NoopMark`?

Reply:
Sure